gr-azure-software-radio/grc/azure_software_radio_keyvault.block.yml

```diff
@@ -21,12 +21,16 @@ parameters:
     label: key
     dtype: string
     default: ''
+-   id: default_cred
+    label: Default Credential
```

Comment:
What do we think about labeling as DefaultAzureCredential here to try to make it more obvious that this isn't just a default username/password?

Reply:
fixed